src/main/java/com/diffblue/deeptestutils/powermock/WhiteboxImpl.java

```diff
@@ -0,0 +1,206 @@
+package com.diffblue.deeptestutils.powermock;
+
+import org.powermock.reflect.exceptions.TooManyMethodsFoundException;
+
+import java.lang.reflect.Method;
+import java.util.Collections;
+import java.util.LinkedList;
+import java.util.List;
+
+// Copyright 2016-2018 Diffblue limited. All rights reserved.
```

Comment:
Should be the first line

Reply:
I put it after the package line as this is how it is done in the rest of the deeptest-utils files.

Reply:
This is how it appears in the rest of the deeptest-utils files, would you still prefer to move it to the first line?